src/UBAFeeCalculator/UBAFeeSpokeCalculator.ts

```diff
@@ -167,61 +172,82 @@ export default class UBAFeeSpokeCalculator {
     return this.config.getBalanceTriggerThreshold(this.chainId, this.symbol);
   }
 
+  /**
+   * A convenience method for resolving the event fee for the spoke and the given symbol and a given flow type
+   * @param amount The amount of tokens to simulate
+   * @param flowType The flow type to simulate
+   * @param flowRange The range of the flow to simulate the event for. Defaults to undefined to simulate the event for the entire flow.
+   * @returns The event fee for the spoke and the given symbol and a given flow type
+   */
+  protected getEventFee(amount: BigNumber, flowType: "inflow" | "outflow", flowRange?: UBAFlowRange): FlowFee {
+    // The rough psuedocode for this function is as follows:
+    // We'll need two inflow/outflow curves
+    // We need to determine which flow curve to use based on the flow type
+    // Compute first balancing fee <- f(x, x+amnt)
+    // Compute second balance fee (oportunity cost) <- g(x+amnt, x)
```

Comment:
Just a minor terminology nit: we're overloading the term balancing fee here a little. We're using it for both the component of the deposit/relay fee that does not go to LPs, but we're also using it for the full fee (the value of the curve).

Put differently, we're describing it as:
balancing_fee = balancing_fee - lp_fee

But these two balancing fees refer to different numbers, which makes it somewhat confusing.